### PR TITLE
Make sure to delete custom domain DNS secret after running tests

### PR DESCRIPTION
**Pull request by videlov**
Remove minikube testcase in test_helpers

**Description**

Changes proposed in this pull request:

- Do not leave credentials when running custom-domain-test
- Remove support for Minikube in test_helpers and also get rid of the dependency to kyma/common

**Related issue(s)**

### CODE REVIEW

File: tests/components/api-gateway/custom_domain_test_env.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  >&2 echo "Environment variable TEST_CUSTOM_DOMAIN is required but not set"
+  exit 2
+fi
+
+if [[ -z ${TEST_SA_ACCESS_KEY_PATH} ]]; then
+  >&2 echo "Environment variable TEST_SA_ACCESS_KEY_PATH is required but not set"
+  exit 2
+fi
+
+export TEST_HYDRA_ADDRESS="https://oauth2.${TEST_DOMAIN}"
```
[the remark this answers is not in the document]
Comment (videlov) [reply tests/components/api-gateway/custom_domain_test_env.sh:20]: absolutely, `make setup-custom-domain` is removed

**Review by triffer — DISMISSED** [review]

File: tests/components/api-gateway/README.md [hunk 2]
```diff
@@ -32,21 +32,15 @@ This command creates a k3d cluster on your local machine, installs Kyma on it, a
...
-
-- `TEST_CUSTOM_DOMAIN` - set this environment variable with your desired custom domain.
-- `TEST_DOMAIN` - set this environment variable with your installed by default Kyma domain.
-
-After exporting these domains, run `make setup-custom-domain` to finish the default test setup.
+If you are using Gardener, make sure that your Kubernetes cluster has the `shoot-cert-service` and `shoot-dns-service` extensions enabled. The desired shoot specification is mentioned in the description of this [issue](https://github.com/kyma-project/control-plane/issues/875).
+Obtain a service account access key with permissions to maintain custom domain DNS entries and export it as a JSON file. To learn how to do it, follow this [guide](https://cloud.google.com/iam/docs/keys-create-delete).
 
+Set the following environment variables:
+- `TEST_DOMAIN` - your default Kyma domain, e.g. `c1f643b.stage.kyma.ondemand.com`
```
Comment (nataliasitko) [inline tests/components/api-gateway/README.md:41]:
```suggestion
- `TEST_DOMAIN` - your default Kyma domain, for example, `c1f643b.stage.kyma.ondemand.com`
```

File: tests/components/api-gateway/README.md [hunk 3]
```diff
@@ -32,21 +32,15 @@ This command creates a k3d cluster on your local machine, installs Kyma on it, a
...
+- `TEST_CUSTOM_DOMAIN` - your custom domain, e.g. `custom.domain.build.kyma-project.io`
```
Comment (nataliasitko) [inline tests/components/api-gateway/README.md:42]:
```suggestion
- `TEST_CUSTOM_DOMAIN` - your custom domain, for example, `custom.domain.build.kyma-project.io`
```

File: tests/components/api-gateway/README.md [hunk 4]
```diff
@@ -32,21 +32,15 @@ This command creates a k3d cluster on your local machine, installs Kyma on it, a
...
+- `TEST_SA_ACCESS_KEY_PATH` - the path to the service account access key exported as a JSON file, e.g. `/Users/user/gcp/service-account.json`
```
Comment (nataliasitko) [inline tests/components/api-gateway/README.md:43]:
```suggestion
- `TEST_SA_ACCESS_KEY_PATH` - the path to the service account access key exported as a JSON file, for example, `/Users/user/gcp/service-account.json`
```

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**